### ISSUE TITLE
add people worth following suggestions to primary feed. 

### ISSUE THREAD

**Issue by rabble** [planetary-social/planetary-ios#452]
![Screen Shot 2022-03-25 at 8 20 40 PM](https://user-images.githubusercontent.com/76/160073531-0ed63b6a-85e5-4eab-b6de-b9fe82b1598b.png)

Every x posts in a users primary feed, we should be suggesting they follow a new person. I'm not sure what the suggestion algorithm should be. Don't worry too much, anything is probably good enough.

Make sure people suggested have updated content within the last month.

Find people who are followed by people you follow. 

Don't repeat suggestions too often...

Basically, even bad suggestions are ok. As long as those users are updating.

If the hashtags are at AT ALL HARD, then put their profile bio in that spot instead. 

Talk to @Chardot before implimenting.

**Comment by martindsq** [comment]
this is fixed with the follow messages, not directly needed for now.
